<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiningPage</class>
 <widget class="QDialog" name="MiningPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="warningMSG">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;div&gt;&lt;span style=&quot;color:#ff0000;&quot;&gt;M&lt;/span&gt;&lt;span style=&quot;color:#f90505;&quot;&gt;i&lt;/span&gt;&lt;span style=&quot;color:#f30b0a;&quot;&gt;n&lt;/span&gt;&lt;span style=&quot;color:#ee100f;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot;color:#e81614;&quot;&gt; &lt;/span&gt;&lt;span style=&quot;color:#e21b19;&quot;&gt;t&lt;/span&gt;&lt;span style=&quot;color:#dc201e;&quot;&gt;o&lt;/span&gt;&lt;span style=&quot;color:#d72623;&quot;&gt; &lt;/span&gt;&lt;span style=&quot;color:#d12b28;&quot;&gt;h&lt;/span&gt;&lt;span style=&quot;color:#cb312d;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot;color:#c53632;&quot;&gt;l&lt;/span&gt;&lt;span style=&quot;color:#bf3b38;&quot;&gt;p&lt;/span&gt;&lt;span style=&quot;color:#ba413d;&quot;&gt; &lt;/span&gt;&lt;span style=&quot;color:#b44642;&quot;&gt;k&lt;/span&gt;&lt;span style=&quot;color:#ae4c47;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot;color:#a8514c;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot;color:#a35751;&quot;&gt;p&lt;/span&gt;&lt;span style=&quot;color:#9d5c56;&quot;&gt; &lt;/span&gt;&lt;span style=&quot;color:#97615b;&quot;&gt;t&lt;/span&gt;&lt;span style=&quot;color:#916760;&quot;&gt;h&lt;/span&gt;&lt;span style=&quot;color:#8c6c65;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot;color:#86726a;&quot;&gt; &lt;/span&gt;&lt;span style=&quot;color:#80776f;&quot;&gt;Q&lt;/span&gt;&lt;span style=&quot;color:#7a7276;&quot;&gt;u&lt;/span&gt;&lt;span style=&quot;color:#746d7d;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot;color:#6e6984;&quot;&gt;r&lt;/span&gt;&lt;span style=&quot;color:#68648a;&quot;&gt;k&lt;/span&gt;&lt;span style=&quot;color:#625f91;&quot;&gt; &lt;/span&gt;&lt;span style=&quot;color:#5b5a98;&quot;&gt;n&lt;/span&gt;&lt;span style=&quot;color:#55569f;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot;color:#4f51a6;&quot;&gt;t&lt;/span&gt;&lt;span style=&quot;color:#494cad;&quot;&gt;w&lt;/span&gt;&lt;span style=&quot;color:#4347b4;&quot;&gt;o&lt;/span&gt;&lt;span style=&quot;color:#3d43ba;&quot;&gt;r&lt;/span&gt;&lt;span style=&quot;color:#373ec1;&quot;&gt;k&lt;/span&gt;&lt;span style=&quot;color:#3139c8;&quot;&gt; &lt;/span&gt;&lt;span style=&quot;color:#2b34cf;&quot;&gt;h&lt;/span&gt;&lt;span style=&quot;color:#2530d6;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot;color:#1e2bdd;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot;color:#1826e4;&quot;&gt;l&lt;/span&gt;&lt;span style=&quot;color:#1221ea;&quot;&gt;t&lt;/span&gt;&lt;span style=&quot;color:#0c1df1;&quot;&gt;h&lt;/span&gt;&lt;span style=&quot;color:#0618f8;&quot;&gt;y&lt;/span&gt;&lt;span style=&quot;color:#0013ff;&quot;&gt;.&lt;/span&gt;&lt;/div&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Available Cores:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Allocated Cores:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLCDNumber" name="coreCountLCD">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLCDNumber" name="allocatedCoresLCD">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="minCoresLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="allocatedCoresSlider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxCoresLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="miningChecked">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../bitcoin.qrc">:/icons/synced</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mineButton">
       <property name="font">
        <font>
         <family>Modern No. 20</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start Mining</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mineStopButton">
       <property name="font">
        <font>
         <family>Modern No. 20</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop Mining</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="miningCrossed">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../bitcoin.qrc">:/icons/quit</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For more effiicent mining, see &lt;a href=&quot;http://www.reddit.com/r/QuarkCoin/comments/1rutja/quarkcoin_mining_guide/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Pool and GPU mining guides&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
